lib/KLabeledIcon.vue

```diff
@@ -29,11 +36,21 @@
       KIcon,
     },
     props: {
-      // If provided, will render a KIcon with the same 'icon' prop
+      // If provided, will render a KIcon using the value given as the `icon` prop. The icon will be prepended to the label.
```

Comment:
nice, thanks for adding the Markdown

```suggestion
      // An icon that will be prepended to the label. Uses the same icon names as the `KIcon` component
```